src/main/java/CommandParser.java

```diff
@@ -0,0 +1,43 @@
+import java.util.HashMap;
+
+public class CommandParser{
+    public static final String ARGUMENT_COMMAND = "command";
+    public static final String ARGUMENT_MAIN = "main";
+    public static HashMap<String, String> parseCommandToArguments(String line) {
+        HashMap<String, String> argumentsList = new HashMap<>();
+        String[] lineArgs = line.split(" ");
+
+        // Command
+        if (lineArgs.length <= 0) {
+            argumentsList.put(CommandParser.ARGUMENT_COMMAND,"");
+            return argumentsList;
+        }
+        argumentsList.put(CommandParser.ARGUMENT_COMMAND,lineArgs[0]);
+
+        // Arguments
+        String currArgumentName = CommandParser.ARGUMENT_MAIN;
+        StringBuilder currArgument = new StringBuilder();
+
+        for (int i=1; i<lineArgs.length; i++) {
+            if (lineArgs[i].isEmpty()) { // Should be redundant but just in case
+                continue;
+            }
+            if (lineArgs[i].charAt(0) == '/') {
+                // New argument
+                if (!currArgument.toString().isEmpty()){
+                    argumentsList.put(currArgumentName, currArgument.toString().strip());
+                }
+                currArgumentName = lineArgs[i];
+                currArgument.setLength(0);
+            } else {
+                // Add on to existing argument
+                currArgument.append(" ").append(lineArgs[i]);
+            }
+        }
+        // Add last command
+        if (!currArgument.toString().isEmpty()) {
+            argumentsList.put(currArgumentName, currArgument.toString().strip());
+        }
+        return argumentsList;
+    }
+}
```

Comment:
I like how your implementation for this class, as it would allow more commands to be added in the future.